robustness/datasets.py

```diff
@@ -88,7 +88,7 @@ def override_args(self, default_args, new_args):
         if len(extra_args) > 0: raise ValueError(f"Invalid arguments: {extra_args}")
         for k in kwargs:
             req_type = type(default_args[k])
-            if not isinstance(kwargs[k], req_type):
+            if (default_args[k] is not None) and (not isinstance(kwargs[k], req_type)):
```

Comment:
Should we also check if  `kwargs[k] is not None` ?
```python
if (default_args[k] is not None and kwargs[k] is not None) and (not isinstance(kwargs[k], req_type)):
````

Reply:
you mean in the case someone wants to make a default argument from something into None? I guess it could be good for future compatibility, will make the change 👍

Reply:
yes exactly

Reply:
@Hadisalman check now and merge if it looks good?

Reply:
Looks good!